test/models/playlist_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class PlaylistTest < ActiveSupport::TestCase
+  test 'valid if danceanbility beetween 1 and 100' do
```

Comment:
De manière général c'est mieux d'écrire une phrase plus "anglaise":

```suggestion
  test 'should have a danceanbility between 1 and 100' do
```

ce commentaire est valide pour tous les autres tests